### PR TITLE
Remove term

### PR DESCRIPTION
**Pull request by majecty**
We are removing CodeChain staking code from Foundry.
This PR does not remove all staking code. This PR is removing term related code.
I'll remove other staking code in following PRs.

### CODE REVIEW

**Comment by junha1** [comment]
I accidentally made a merge commit. Would you remove that and rebase it onto the latest master?

**Comment by majecty** [comment]
@junha1 Did you click "Update branch"(I'm not sure) in GitHub?
The merge commit made by GitHub will be removed when this PR is merged using "Rebase and merge" button.

**Comment by junha1** [comment]
@majecty Yes I did. And I was talking about two commits 4fcc819 and 0fe4ede, which are already merged in the previous PR.

File: test/src/e2e.dynval/1/doubleVote.test.ts [hunk 1]
```diff
@@ -1,134 +0,0 @@
-// Copyright 2019 Kodebox, Inc.
```
Comment (junha1) [inline test/src/e2e.dynval/1/doubleVote.test.ts:1]: What about `test/src/e2e/termChange.test.ts`?

File: spec/Staking.md [hunk 2]
```diff
@@ -1,412 +0,0 @@
-Anyone who holds staking tokens called 'CCS' can share fees that are paid for transactions.
```
Comment (junha1) [inline spec/Staking.md:1]: This too `spec/Dynamic-Validator.md`

File: test/src/e2e.dynval/1/doubleVote.test.ts (hunk 1 above)
Comment (majecty) [reply test/src/e2e.dynval/1/doubleVote.test.ts:1]: Thanks. I removed it.

File: spec/Staking.md (hunk 2 above)
Comment (majecty) [reply spec/Staking.md:1]: Thanks. I removed it.

**Review by junha1 — APPROVED** [review]

**Comment by junha1** [comment]
Could you check the CI errors?

**Comment by majecty** [comment]
@junha1 Which CI error are you saying?
Should PRs succeed to run unit tests?

**Comment by junha1** [comment]
@majecty Yes. My last PR has succeeded on unit tests. (Your previous PR failed on Mac because of resource limit).

**Comment by majecty** [comment]
I removed unit tests from branch protection rule before. I thought the unit tests are unstable in the master branch.
I'll add unit tests to the branch protection rule.

**Review by junha1 — APPROVED** [review]

**Pull request merged**